extensions/markdown-language-features/preview-src/index.ts

```diff
@@ -37,32 +37,51 @@ window.onload = () => {
 	updateImageSizes();
 };
 
+
+function doAfterImagesLoaded(cb: () => void) {
+	const imgElements = document.getElementsByTagName('img');
+	if (imgElements.length > 0) {
+		const ps = Array.from(imgElements).map(e => {
+			if (e.complete) {
+				return Promise.resolve();
+			} else {
+				return new Promise<void>((resolve) => {
+					e.addEventListener('load', () => resolve());
```

Comment:
Instead of checking all images, should this use the [`window.load`](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) event? That should only be fired when all resources are loaded

If we continue checking for each image, should there also be a check for the `error` event in case the image fails to load?

Reply:
Using `window.addEventListener('load', cb);` does not work. When `webview.html = content` executed on the extension host, the `load` event does not occur on the webview.

I have edited the PR to treat cases when the error event occurs.